plugin/api.go

```diff
@@ -31,6 +31,11 @@ type API interface {
 	// GetConfig fetches the currently persisted config
 	GetConfig() *model.Config
 
+	// GetUnsanitizedConfig fetches the currently persisted config without removing secrets
```

Comment:
missing a period in the end `secrets.` to be godoc compliant https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences